docs/network_graph_attributes.md

```diff
@@ -132,9 +132,9 @@ An optional, human-meaningful string description of the edge. The string may be
 #### `edge.latency`
 
 Required: True  
-Type: Float
+Type: String
 
-The latency that will be added to packets traversing this edge. This value is used as a weight while running Dijkstra's shortest path algorithm.
+The latency that will be added to packets traversing this edge. This value is used as a weight while running Dijkstra's shortest path algorithm. The format of the string specifies the latency and its unit, e.g., `10 ms`. If a unit is not specified, it will be assumed that it is in the base unit of "seconds".
```

Comment:
Do we document anywhere what the acceptable time units are? For example, ms, milliseconds, s, seconds, etc?

I mean time units in general, whenever a time is given (either here or in other parts of the config such as heartbeatinterval).

Reply:
We don't currently. I added this as a todo item in the 2.0 pre.1 project and will do it in a new PR.